pubspec.yaml

```diff
@@ -49,6 +62,8 @@ flutter:
   assets:
     - images/PairingsBg.png
```

Comment:
As we add more images to this project, we should probably [add all assets within a directory](https://flutter.dev/docs/development/ui/assets-and-images#specifying-assets) by:
```yaml
assets:
   - images/
```